whd/pull_request.py

```diff
@@ -420,41 +399,15 @@ def ensure_pr_resource_updates(
     concourse_api,
     pr_event: PullRequestEvent,
     resources: typing.List[concourse.client.model.PipelineConfigResource],
-    es_client: ccc.elasticsearch.ElasticSearchClient,
     retries=10,
     sleep_seconds=3,
 ):
     time.sleep(sleep_seconds)
 
     retries -= 1
     if retries < 0:
-        try:
-            log_outdated_resources(
```

Comment:
will it make sense to keep logging (to stdout/stderr) for potential debugging?

Reply:
I have never used this metric at all, even though it was introduced back in 2019 with https://github.com/gardener/cc-utils/commit/dd4ba15e68 😅 

I don't see value in keeping them.